### PR TITLE
[DC-1038] Create bash script to clean Fitbit data

### PR DESCRIPTION
**Pull request by nishanthpp93**
* Add clean_fitbit.sh to run clean deid rules on fitbit dataset

### CODE REVIEW

File: data_steward/tools/clean_fitbit.sh [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+fitbit_deid_dataset="${dataset_release_tag}_deid"
+registered_fitbit_deid="R${fitbit_deid_dataset}"
+ROOT_DIR=$(git rev-parse --show-toplevel)
+DATA_STEWARD_DIR="${ROOT_DIR}/data_steward"
+TOOLS_DIR="${DATA_STEWARD_DIR}/tools"
+CLEANER_DIR="${DATA_STEWARD_DIR}/cdr_cleaner"
+CLEAN_DEID_DIR="${CLEANER_DIR}/cleaning_rules/deid"
+
+export BIGQUERY_DATASET_ID="${fitbit_dataset}"
```
Comment (ChaoPang) [inline data_steward/tools/clean_fitbit.sh:65]: Do we need to declare this environment variable?

File: data_steward/tools/clean_fitbit.sh [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+# Use the below command if copy fails
+#transfer_params='{"source_dataset_id":"'${registered_fitbit_deid}'","source_project_id":"'${APP_ID}'"'
+#bq mk --transfer_config --project_id="${APP_ID}" --data_source="cross_region_copy" --target_dataset="${registered_fitbit_deid}" --display_name='Create Fitbit Deid' --params="${transfer_params}"
+
+# create empty fitbit sandbox dataset
+sandbox_dataset="${registered_fitbit_deid}_sandbox"
+bq mk --dataset --description "${dataset_release_tag} sandbox dataset for ${registered_fitbit_deid}" "${APP_ID}":"${sandbox_dataset}"
+
+# Apply cleaning rules
+python "${CLEAN_DEID_DIR}/remove_fitbit_data_if_max_age_exceeded.py" --project_id "${APP_ID}" --dataset_id "${registered_fitbit_deid}" --sandbox_dataset_id "${sandbox_dataset}" --combined_dataset_id "${combined_dataset}" -s 2>&1 | tee logs/fitbit_max_age_log.txt
```
Comment (ChaoPang) [inline data_steward/tools/clean_fitbit.sh:80]: I don't think the logs folder exists by default and the output could not be written to the log files.

File: data_steward/tools/clean_fitbit.sh (hunk 1 above)
Comment (nishanthpp93) [reply data_steward/tools/clean_fitbit.sh:65]: Yes, this is set as the default dataset for BigQuery queries.

File: data_steward/tools/clean_fitbit.sh (hunk 2 above)
Comment (nishanthpp93) [reply data_steward/tools/clean_fitbit.sh:80]: Thanks for catching this, I'll need to add mkdir

File: data_steward/tools/clean_fitbit.sh [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+# Use the below command if copy fails
+#transfer_params='{"source_dataset_id":"'${registered_fitbit_deid}'","source_project_id":"'${APP_ID}'"'
+#bq mk --transfer_config --project_id="${APP_ID}" --data_source="cross_region_copy" --target_dataset="${registered_fitbit_deid}" --display_name='Create Fitbit Deid' --params="${transfer_params}"
+
+# create empty fitbit sandbox dataset
+sandbox_dataset="${registered_fitbit_deid}_sandbox"
+bq mk --dataset --description "${dataset_release_tag} sandbox dataset for ${registered_fitbit_deid}" "${APP_ID}":"${sandbox_dataset}"
+
+# Create logs dir
+mkdir -p ../logs
```
Comment (ChaoPang) [inline data_steward/tools/clean_fitbit.sh:80]: Perhaps it's better to use `${ROOT_DIR}/logs` or `$DATA_STEWARD_DIR/logs` instead of a relative path?

**Pull request merged**